src/utils/timeSeriesResultsByOption.js

```diff
@@ -1,147 +1,116 @@
-import * as _ from 'lodash'
-import { FV, PMT } from 'formulajs/lib/financial'
-
 const COMPOUND_FREQUENCY = 12
 
-/**
- * @param {number} option.loanAmt How much money the loan is
- * @param {number} option.investmentRate Market return rate (in decimals)
- * @param {number} option.inflationRate Annual inflation rate (in decimals)
- * @param {object} option.shorterOption
- *   @property {number} mortgageRate Mortgage APR (in decimals)
- *   @property {number} mortgageTerm Mortgage term (in years)
- * @param {object} option.longerOption
- *
- * @return {object}
- *   @property {object[]} shorter
- *     @property {number} pmt Monthly payment amount (for that year)
- *     @property {number} loanAmt Remaining loan amount
- *     @property {number} investmentAmount How much money in investments
- *   @property {object[]} longer
- */
-export function getMonthly ({
+export const getMonthly = ({
   loanAmt,
   investmentRate,
   inflationRate,
   shorterOption,
   longerOption
-}) {
-  const shorterOptionPMT = PMT(
-    shorterOption.mortgageRate / COMPOUND_FREQUENCY,
-    shorterOption.mortgageTerm * COMPOUND_FREQUENCY,
-    loanAmt
-  )
-
-  const longerOptionPMT = PMT(
-    longerOption.mortgageRate / COMPOUND_FREQUENCY,
-    longerOption.mortgageTerm * COMPOUND_FREQUENCY,
-    loanAmt
-  )
-
-  const monthlyResultsByOption = {
-    shorter: [
-      {
-        budget: shorterOptionPMT,
-        pmt: shorterOptionPMT,
-        loanAmt: loanAmt + shorterOptionPMT,
-        investmentAmount: 0
-      }
-    ],
-    longer: [
-      {
-        budget: shorterOptionPMT,
-        pmt: longerOptionPMT,
-        investmentPMT: -1 * (longerOptionPMT - shorterOptionPMT),
-        loanAmt: loanAmt + longerOptionPMT,
-        investmentAmount: -1 * (longerOptionPMT - shorterOptionPMT)
-      }
-    ]
-  }
-
-  for (
-    let month = 1;
-    month <= longerOption.mortgageTerm * COMPOUND_FREQUENCY;
-    month++
-  ) {
-    monthlyResultsByOption.shorter.push(
-      getMonthlyResult(
-        shorterOption.mortgageRate,
-        investmentRate,
-        inflationRate,
-        monthlyResultsByOption.shorter[month - 1]
-      )
-    )
-    monthlyResultsByOption.longer.push(
-      getMonthlyResult(
-        longerOption.mortgageRate,
-        investmentRate,
-        inflationRate,
-        monthlyResultsByOption.longer[month - 1]
-      )
-    )
+}) => {
+  const monthlySeries = {}
+
+  const budget = Math.abs(shorterOption.mortgagePMT)
```

Comment:
sug: This probably isn't needed. Could we just use positives everywhere?

Reply:
shorterOption.mortgagePMT comes through as a negative number. I could negate it again, or negate it when the function is called on index.js, but that won't cover edge cases.

I use Math.abs here because the payment could actually come through on as both negative and positive (if you use a negative loan amt, which we shouldn't allow), and budget always has to be positive.